_scripts/instruction-widget/templates/install/dnsplugins.html

```diff
@@ -1,21 +1,23 @@
+<h4>Installing DNS plugins</h4>
 {{#dns_plugins}}
 <p>
-<strong>Certbot's DNS plugins are also available for your system which can be used to
-automate obtaining a wildcard certificate from Let's Encrypt's ACMEv2 server.</strong>
-To use one of these plugins, you must have configured DNS for the domain you
-want to obtain a certificate for with a DNS provider that Certbot has a plugin
-for. A list of these plugins and more information about using them can be found
-<a href="/docs/using.html#dns-plugins">here</a>. To install one of these
-plugins, run the installation command above but replace {{package}} with the
-name of the DNS plugin you want to install.
+Certbot's DNS plugins <strong>are available</strong> for your system. These
+plugins can be used to automate obtaining a wildcard certificate from Let's
+Encrypt's ACMEv2 server.  To use one of these plugins, you must have configured
+DNS for the domain you want to obtain a certificate for with a DNS provider
+that Certbot has a plugin for. A list of these plugins and more information
+about using them can be found <a href="/docs/using.html#dns-plugins">here</a>.
+To install one of these plugins, run the installation command above but replace
+<tt>{{package}}</tt> with <tt>{{dns_package_prefix}}-PLUGIN</tt>, where <tt>PLUGIN</tt>
+is the name of the plugin you want to install.
```

Comment:
I'm worried that people will think the name of the plugin is "certbot-dns-whatever" because that's what's listed at https://certbot.eff.org/docs/using.html#dns-plugins. They'll probably realize they shouldn't try and install something like "python-certbot-dns-certbot-dns-cloudflare", but can we better clarify what we mean here?

Reply:
I added an example using the `rfc2136` plugin!